### PR TITLE
Update module metadata after transfer to Voxpupuli

### PR DESCRIPTION
**Pull request by isc-lindensc**
fix metadata - dependency and location referenced

### CODE REVIEW

**Review by smortex — APPROVED** [review]
LGTM!

**Comment by ekohl** [comment]
I think https://github.com/voxpupuli/puppet-augeasproviders_sysctl/pull/65 is intended to really make it a VP module.

File: metadata.json [hunk 1]
```diff
@@ -1,16 +1,16 @@
...
-  "source": "https://github.com/hercules-team/augeasproviders_sysctl",
-  "project_page": "https://github.com/hercules-team/augeasproviders_sysctl",
-  "issues_url": "https://github.com/hercules-team/augeasproviders_sysctl/issues",
+  "source": "https://github.com/voxpupuli/puppet-augeasproviders_sysctl",
+  "project_page": "https://github.com/voxpupuli/puppet-augeasproviders_sysctl",
+  "issues_url": "https://github.com/voxpupuli/puppet-augeasproviders_sysctl/issues",
   "description": "This module provides type/provider for sysctl using the Augeas configuration API library.",
   "dependencies": [
     {
-      "name": "herculesteam/augeasproviders_core",
```
Comment (deric) [inline metadata.json:13]: Just bear in mind that this means bumping dependency to:
```
"version_requirement": ">= 3.2.0 < 4.0.0"
```
There's no other release of [puppet/augeasproviders_core](https://forge.puppet.com/modules/puppet/augeasproviders_core/readme) for backwards compatibility.

Comment (deric) [reply metadata.json:13]:
```
Module herculesteam-augeasproviders_core found versions: 3.1.0, 3.0.0, 2.7.0, 2.6.0, 2.5.0, 2.4.0, 2.3.0, 2.2.0, 2.1.5, 2.1.4, 2.1.3, 2.1.2, 2.1.1, 2.1.0, 2.0.1, 2.0.0
```
librarian seems to be able to deal with this change:
```
Dependency on module 'augeasproviders_core' is fullfilled by multiple modules and only one will be used: ["herculesteam-augeasproviders_core", "puppet-augeasproviders_core"]
```

**Comment by deric** [comment]
Please merge this and release with a new  `puppet-augeasproviders_sysctl` version with tag, e.g.  `3.0.0` so that other modules can update their dependencies. The existence of `puppet-augeasproviders_core` module while no module `puppet-augeasproviders_sysctl` exists, is confusing.

**Comment by ekohl** [comment]
I've updated #65 to include your change. Somehow idempotency appears broken with Puppet 7 in one specific case, which I don't understand yet.

**Comment by ekohl** [comment]
I am stuck on https://github.com/voxpupuli/puppet-augeasproviders_sysctl/pull/65#issuecomment-1423868243 and really hesitant to merge something broken. Note that on your PR no CI ran at all, so it may look green but it just didn't test anything.

**Comment by jcpunk** [comment]
It looks like Augeas 1.14 has some fixes for the sysctl lense, I wonder if that is relevant?

**Comment by deric** [comment]
@jcpunk The current issues with failing tests isn't probably related to changes in Augeas. [This PR](https://github.com/hercules-team/augeas/pull/755) probably won't affect this module. I haven't seen any breaking changes, and the augeas package usually installed from OS package, right?

**Comment by h-haaks** [comment]
@isc-lindensc could you please rebase this against master now that #65 is merged?

**Comment by h-haaks** [comment]
@isc-lindensc I'll close this PR since there is no more changes to apply.

**Pull request closed without merging**